glados_respeaker.py

```diff
@@ -19,13 +19,12 @@ def respeaker_errors(e):
 		exit();
 	if "name 'pixel_ring' is not defined" in str(e):
 		print("\nERROR: ReSpeaker is probably not connected?")

```

Comment:
I think this line should be below the check for variable. It's pointless to mention ReSpeaker is not connected if it is disabled in config on purpose.

Reply:
With the if statements fixed, the ones checking the settings.env, respeaker_errors() should never get called. Therefore this error mesase is still good if it’s enabled in config, but unplugged or not working. 

Also, I believe the variable checks in glados_functions.py are redundant, I’ll look into that more in a bit.